cv19ResSupportV3.Tests/V3/Factories/Commands/CreateHelpRequestCallFactoryTest.cs

```diff
@@ -0,0 +1,35 @@
+using AutoFixture;
+using cv19ResSupportV3.V3.Boundary.Requests;
+using cv19ResSupportV3.V3.Domain.Commands;
+using cv19ResSupportV3.V3.Factories.Commands;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace cv19ResSupportV3.Tests.V3.Factories.Commands
+{
+    [TestFixture]
+    public class CreateHelpRequestCallFactoryTest
+    {
+
+        [Test]
+        public void RequestToCommand()
+        {
+            var request = new Fixture().Build<CreateHelpRequestCallRequest>().Create();
```

Comment:
I'm pretty sure you can do:
`new Fixture().Create<CreateHelpRequestCallRequest>();`
You only need the `.Build<T>()` when you want exclude some fields from being generated or when you want to control the values yourself for some of the fields.